dask/dataframe/core.py

```diff
@@ -2394,13 +2436,19 @@ def info(self, buf=None, verbose=False, memory_usage=False):
 for name in ['add', 'sub', 'mul', 'div',
              'truediv', 'floordiv', 'mod', 'pow',
              'radd', 'rsub', 'rmul', 'rdiv',
-             'rtruediv', 'rfloordiv', 'rmod', 'rpow']:
+             'rtruediv', 'rfloordiv', 'rmod', 'rpow', 'eq']:
```

Comment:
Why is only `eq` added here and not the others?  In what was is it special?

To be clear, I'm genuinely ignorant about how a lot of dask.dataframe is organized these days.  This is a genuine question, and not a potentiall snarky comment :)